node/grpc/v2_metrics.go

```diff
@@ -0,0 +1,175 @@
+package grpc
+
+import (
+	"github.com/Layr-Labs/eigensdk-go/logging"
+	grpcprom "github.com/grpc-ecosystem/go-grpc-middleware/providers/prometheus"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/collectors"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+	"google.golang.org/grpc"
+	"os"
+	"path/filepath"
+	"sync/atomic"
+	"time"
+)
+
+const namespace = "eigenda_node"
+
+// V2Metrics encapsulates metrics for the v2 DA node.
+type V2Metrics struct {
```

Comment:
MetricsV2? It's in "XxxV2" format for other components

Reply:
renamed